ais/models.py

```diff
@@ -852,6 +863,10 @@ class AddressSummary(db.Model):
     geocode_type = db.Column(db.Text)
     geocode_x = db.Column(db.Float)
     geocode_y = db.Column(db.Float)
+    # geocode_curb_x = db.Column(db.Float)
+    # geocode_curb_y = db.Column(db.Float)
+    geocode_street_x = db.Column(db.Float)
```

Comment:
Ok, I'll test that approach - another option is to include the geocode values in address_summary as a dictionary in one field...